### DISCUSSION TITLE
Comfy Ui error with Animate Diff

### DISCUSSION THREAD

**Discussion by Bilboa-saggins** [Comfy-Org/ComfyUI#2092] [Q&A]
keep getting this error no matter what i do

![image](https://github.com/comfyanonymous/ComfyUI/assets/145880723/4cb44fc7-aadb-45df-ae4c-315c52470249)

**Comment by ltdrdata** [comment]
Both ComfyUI and Animatediff need to be updated to the latest versions. However, due to the lack of updates for Animatediff in the past two weeks, it may not be properly supported. It is recommended to use Animatediff-Evolved instead.
